### DISCUSSION TITLE
Input date and time

### DISCUSSION THREAD

**Discussion by kschmott** [saadeghi/daisyui#1778] [Ideas / feature requests]
Title. Would be nice to have these in daisy ui.

**Comment by saadeghi** [comment]
You can use `input` class to style the `<input>` itself but unfortunately the native selector drop down is impossible to be styled using CSS. Libraries you see with customized datepicker style, are using JS to simulate a datepicker with divs and spans.  
However there are datepicker JS plugins with optional styles where you can apply daisyUI colors to them.
